go/updater/updater.go

```diff
@@ -307,7 +310,10 @@ func (u *Updater) apply(src string, dest string) (tmpPath string, err error) {
 
 // Update checks, downloads and performs an update.
 func (u *Updater) Update(ui libkb.UpdateUI, force bool, requested bool) (update *keybase1.Update, err error) {
-	update, err = u.CheckForUpdate(false, force, requested)
+	u.Lock()
+	defer u.Unlock()
```

Comment:
Oh it's in the checker here:
https://github.com/keybase/client/blob/updatechecktweaks/go/updater/update_checker.go#L41-L54

Which is actually outside the Update(..) method (in the checker ticker). In which case to be defensive, I should add a running check and return first if already running?